sympy/simplify/simplify.py

```diff
@@ -994,7 +994,16 @@ def goodlog(l):
                 num, den = k, -k
                 if num.count_ops() > den.count_ops():
                     num, den = den, num
-                other.append(num*log(log1.pop(num).args[0]/log1.pop(den).args[0]))
+                abc = log1.pop(num).args[0]
+                pqr = log1.pop(den).args[0]
+                if (abc.is_Number) and (pqr.is_Number) :
+                    abc1 = int(abc)
```

Comment:
Note that this will change the value of `abc` if it is not an integer.

Reply:
@jksuom but if abc is not an integer in the first place, it will not enter the if statement

Reply:
And it is working perfectly fine

Reply:
If we take the logarithm of a non-integer number like 3/2 the result is not correct.
```
>>> a = log(Rational(3, 2), evaluate=False)
>>> a
log(3/2)
>>> logcombine(a - log(2))
log(-log(2)/2)  # log(3/4) is expected
```

Reply:
@jksuom I have figured out how to solve logcombine(log(2) - log(Rational(3,2) , evaluate = False)), can you suggest how to proceed to solve logcombine(log(Rational(3,2) , evaluate = False) , log(2)) ?